replica.yml

```diff
@@ -1,13 +1,11 @@
 ---
-version: "3"
+version: "3.4"
 services:
   data-transport-layer:
     image: ethereumoptimism/data-transport-layer:${DTL_IMAGE_TAG:-latest}
     env_file:
       - ${SHARED_ENV_PATH}/data-transport-layer.env
       - .env
-    ports:
-      - ${DTL_PORT:-7878}:7878
```

Comment:
Whats the reasoning behind removing the DTL port? 

When helping users troubleshoot issues with the replica, calling DTL endpoints is very useful.

Reply:
Ah interesting. I saw no point to it. Let me move it over to the shared yml, one sec.